src/System.Memory/tests/ReadOnlySpan/GcReporting.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static class ReadOnlySpanGcReportingTests
+    {
+        [Fact]
+        public static void DelegateTest()
+        {
```

Comment:
It may be useful to add some comment about what this test is about: It is a simple sanity test to check that GC reporting for Span is not completely broken, it is not meant to be comprehensive.